### DISCUSSION TITLE
Warning about forcing a new head silently

### DISCUSSION THREAD

**Discussion by raojianli** [filecoin-project/lotus#9621] [Build and Sync]
WARN	chainstore	store/store.go:523	(!!!) forcing a new head silently; new head: [bafy2bzacedn3nihmzuxydawzsxe22ur3em47dml26lr3jevfkuowplqfziw2e bafy2bzaceb4b2hue2wngbpnvki6lhj35zfp4wgh7ffpzze2ldq2y2pi4lrwvu bafy2bzacebspnbyoqky6qozwzftn5zkblkhs5jtxacmsrmu2ftaermakwy3dm bafy2bzacebyywgcsztojcxsedv73ib5gs5y2kde2e3rhs3mhljtamydw56r3m]

**Comment by rjan90** [comment]
Hey @raojianli!

Can you elaborate a bit more what you are doing when you got this error?

**Comment by LexLuthr** [comment]
ForceHeadSilent() is called only when importing snapshots. Are you seeing it in the logs when importing a snapshot?
